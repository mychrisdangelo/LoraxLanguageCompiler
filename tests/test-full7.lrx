/* 
 * Author: Zhaarn Maheswaran
 * End to end test of tree passing, function call
 */

int test_tree(tree <int>t(2))
{
	print(t);
	return 0;
}

int main()
{
	tree <int>t(2);
	t = 1[2, 3[4, 5]];
	test_tree(t);
}
