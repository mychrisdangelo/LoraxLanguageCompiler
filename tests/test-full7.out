1[2[null,null],3[4[null,null],5[null,null]]]